packages/block-library/src/image/edit.native.js

```diff
@@ -116,9 +118,11 @@ class ImageEdit extends React.Component {
 		} else if ( attributes.id && ! isURL( attributes.url ) ) {
 			requestImageFailedRetryDialog( attributes.id );
 		}
-
+		// eslint-disable-next-line no-undef
+		const enableFullscreen = __DEV__;
```

Comment:
We should cast it to a boolean here to avoid the `enableFullscreen && true` trick